src/unix.rs

```diff
@@ -613,9 +628,17 @@ pub fn set_thread_priority_and_policy(
 
                 // Normal priority threads adjust relative priority through niceness.
                 set_errno(0);
-                let ret = unsafe { libc::setpriority(libc::PRIO_PROCESS, 0, fixed_priority) };
-                if ret != 0 {
-                    return Err(Error::OS(errno()));
+                // VxWorks does not have setpriority function call. Control never
```

Comment:
Can you add a blank line to separate the `set_errno(0);` and the following vxworks-related code, please?